include/mbedtls/mbedtls_config.h

```diff
@@ -2065,6 +2065,25 @@
  */
 #define MBEDTLS_AESNI_C
 
+/**
+ * \def MBEDTLS_AESCE_C
+ *
+ * Enable AES crypto engine support on Arm64.
```

Comment:
Commonly, when we say crypto engine, it represent one indepent hw crypto engine. Here we are make using of the crypto extentions of CPU, so, should we change the word `engine `to `extention`?

Reply:
Sure, will change that